src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -1093,6 +1101,63 @@ private void EmitSearchSelectionTelemetry(PackageItemViewModel selectedPackage)
             }
         }
 
+        private void IncrementInstalledPackageSelectionCount()
+        {
+            PackageItemViewModel selectedItem = _packageList.SelectedItem;
```

Comment:
UI thread check to avoid double counting:

```suggestion
            ThreadHelper.ThrowIfNotOnUIThread();
            PackageItemViewModel selectedItem = _packageList.SelectedItem;
```

Reply:
Same as above, can you help me understand how double counting would occur?

Reply:
DIscussed offline. We don't need UI thread check.